docs/deploy.rst

```diff
@@ -0,0 +1,244 @@
+.. _deploy:
+
+
+=========================
+Deploy your first cluster
+=========================
+
+This page shows you how to:
+
+- Create an organization
+- Set a region in Croud (determines where your data is stored)
+- Create a project
+- Check for available CrateDB products in that region
+- Deploy your first CrateDB cluster
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+.. NOTE::
+
+  This document assumes that you're `signed in`_ to your CrateDB Cloud user
+  account.
+
+
+Create an organization
+----------------------
+
+To begin with, let's create an organization. Organizations hold projects
+that let you deploy CrateDB products such as `clusters`_. Once you've
+created an organization, you can invite CrateDB Cloud users to become
+`organization members`_.
+
+Per default, you will be assigned as the administrator of an organization that
+you create.
+
+Give the `croud organizations create`_ command and pass in ``--name`` to specify
+the organization name along with ``--plan-type`` to determine the support plan
+you want to use, like so:
+
+.. code-block:: console
+
+    sh$ croud organizations create --name YourOrganizationName --plan-type 1
+
+Use the ``croud organizations list`` command to verify that you are a member of
+the organization you created:
+
+.. code-block:: console
+
+    sh$ croud organizations list
+    +--------------------------------------+----------------+-------------+
+    | id                                   | name           |   plan_type |
+    |--------------------------------------+----------------+-------------|
+    | 59c7961b-33a2-4478-9e98-2a6b3f54f60e | MyOrganization |           1 |
+    +--------------------------------------+----------------+-------------+
+
+This will output a list of organizations that hold you as a member. The
+command's output includes the organization's org-ids, names, and plan types.
+
+
+Create a project
+----------------
+
+Next, you should set up a region. The ``region`` setting determines where your
+data is stored. It also specifies which CrateDB products, such as clusters, you
+will find available.
+
+The following regions are available: Azure East-US, Azure West-Europe, Bregenz.
+
+These are your possible choices for the ``region`` flag in the ``croud config``
+command:
+
+- eastus.azure
+- eastus2.azure
+- westeurope.azure
+- bregenz.a1
+
+This tutorial uses the region ``bregenz.a1``.
+
+Run the ``croud config`` command and pass in ``region`` to set a region, like
+so:
+
+  .. code-block:: console
+
+     sh$ croud config set --region bregenz.a1
+     ==> Info: Region switched to bregenz.a1
+
+Projects contain CrateDB products such as clusters and are created in specific
+regions. A CrateDB product that you deploy in a specific project is tied to its
+region.
+
```

Comment:
Maybe we should note that every command the user now executes will be executed for the "bregenz.a1" region. However, they can always override this setting by passing the `-r ` parameter to the command.
So if they e.g. run `croud clusters list` (default bregenz.a1) and wonder why they didn't get any results (they previously deployed a cluster in eastus.azure) we should point out that it is due to querying the wrong region